services/kommander-appmanagement/0.16.0/kommander-appmanagement.yaml

```diff
@@ -1,18 +1,25 @@
 ---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: OCIRepository
+metadata:
+  name: kommander-appmanagement-chart
+  namespace: kommander-flux
```

Comment:
I'm wondering if it should go to the `${releaseNamespace}`

Reply:
done